source/controlTypes.py

```diff
@@ -613,6 +613,23 @@
 REASON_ONLYCACHE="onlyCache"
 #}
 
+# Text to use for 'current' values. These describe if an item is the current item 
+# within a particular kind of selection.

```

Comment:
Please change the comment prefix to `#:`:

```
#: l1
#: l2
```

This means it'll be treated as the documentation of the variable by epydoc (the code doc tool).